Baby Monitor/Source Files/Services/Websocket/WebSocketServerProtocol.swift

```diff
@@ -60,27 +63,38 @@ final class PSWebSocketServerWrapper: NSObject, WebSocketServerProtocol {
 }
 
 extension PSWebSocketServerWrapper: PSWebSocketServerDelegate {
-    func serverDidStart(_ server: PSWebSocketServer) {}
+    func serverDidStart(_ server: PSWebSocketServer) {
+        connectionStatusPublisher.onNext(.connecting)
+    }
     
-    func serverDidStop(_ server: PSWebSocketServer) {}
+    func serverDidStop(_ server: PSWebSocketServer) {
+        connectionStatusPublisher.onNext(.disconnected)
+    }
+    
+    func server(_ server: PSWebSocketServer, didFailWithError error: Error) {
+        connectionStatusPublisher.onNext(.disconnected)
+    }
     
-    func server(_ server: PSWebSocketServer, didFailWithError error: Error) {}
     
     func server(_ server: PSWebSocketServer, webSocketDidOpen webSocket: PSWebSocket) {
         connectedSocketPublisher.accept(PSWebSocketWrapper(socket: webSocket, assignDelegate: false))
+        connectionStatusPublisher.onNext(.connected)
     }
     
     func server(_ server: PSWebSocketServer, webSocket: PSWebSocket, didReceiveMessage message: Any) {
         guard let decodableMessage = message as? WebsocketMessageDecodable,
-            let stringMessage = decodableMessage.decode() else {
-                return
+              let stringMessage = decodableMessage.decode() else {
```

Comment:
Wrong indentation.

Reply:
?? all `let` in guard statemant are aligned...

Reply:
Hm, what if you will click enter in Xcode that will be the default alignment?

Reply:
I can't really tell - I don't usually use XCode for coding...

Reply:
Oh, I've checked and the version before is the default one :)

Reply:
I don't think it's all about which IDE we choose to use - it's rather about defining common styleguide. I asked my Mentor to send me screenshots of XCode styleguide settings in his project and applied changes to my AppCode styleguide. IMHO aligning `let` in `guard` statement is a good thing, and if XCode doesn;t have it by default, we should add this rule.